pkgs/development/compilers/gcc/common/dependencies.nix

```diff
@@ -88,5 +85,8 @@ in
   ;
 
   # threadsCross.package after gcc6 so i assume its okay for 4.8 and 4.9 too
-  depsTargetTarget = optionals (!withoutTargetLibc && threadsCross != { } && threadsCross.package != null) [ threadsCross.package ];
+  depsTargetTarget = optionals (!withoutTargetLibc && threadsCross != { } && threadsCross.package != null) [ threadsCross.package ]
+  ++ [
+    stdenv.cc.bintools # For linking code at run-time
```

Comment:
```
nix-repl> pkgsCross.aarch64-multiplatform.__splicedPackages.stdenv.cc.bintools ? __spliced
false
```

Reply:
I think it's okay; nothing uses `stdenv.cc.bintools` as a dependency AFAICT -- it's mainly there "for reference purposes".

Put another way,

```
nix-instantiate . -A pkgsCross.aarch64-multiplatform.bash
```

emits no warnings.

Reply:
Latest push adds a `targetPackages_bintools` to the top level, which is basically just `targetPackages.stdenv.cc.bintools` except that it gets spliced.